features/term-management-by-slug.feature

```diff
@@ -0,0 +1,88 @@
+Feature: Manage WordPress terms by slug or ID
+
+  Background:
+    Given a WP install
+
+  Scenario: Deleting a term by slog or ID
+    When I run `wp term create category Apple --description="A type of fruit"`
+    Then STDOUT should be:
+    """
+    Success: Created category 2.
+    """
```

Comment:
This and other statements should be indented with two spaces, as they are further down:

```
  Then STDOUT should be:
    """
    Success: Created category 2.
    """
```